### PR TITLE
#63 #92 Content is scrollable when one tab is longer

### PR DESCRIPTION
**Pull request by pi0trpietruszewski**
### Description

- Removed scrollToTop from ScrollableTabView
- Added scrollToTop in componentDidUpdate to StickyParallaxHeader
- Added minHeight to tab containers and webviews
- Added maxHeight equal to full container height to inactive tab

### CODE REVIEW

File: src/components/ScrollableTabView/ScrollableTabView.js [hunk 1]
```diff
@@ -104,12 +92,17 @@ class ScrollableTabView extends React.Component {
     this.children().map((child, idx) => {
       const key = this.makeSceneKey(child, idx)
       const { currentPage, containerWidth, sceneKeys } = this.state
+      const {minViewportHeight } = this.props
```
Comment (IdaszakDaniel) [inline src/components/ScrollableTabView/ScrollableTabView.js:95]: I think that linter somehow missed this space before minViewportHeight

File: src/StickyParallaxHeader.js [hunk 2]
```diff
@@ -44,6 +44,20 @@ class StickyParallaxHeader extends Component {
     this.props.onRef?.(this)
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    const { headerHeight, parallaxHeight, tabs } = this.props
+    const { currentPage, isFolded } = this.state
+    const isRenderingTabs = tabs && tabs.length > 0
+
+    if (isRenderingTabs && prevState.currentPage !== currentPage && isFolded) {
```
Comment (IdaszakDaniel) [inline src/StickyParallaxHeader.js:52]: Would be great if you could also export this currentPage check to variable 👌

Comment (pi0trpietruszewski) [reply src/StickyParallaxHeader.js]: exported to external variable

File: src/components/ScrollableTabView/ScrollableTabView.js (hunk 1 above)
Comment (pi0trpietruszewski) [reply src/components/ScrollableTabView/ScrollableTabView.js]: fixed

**Pull request merged**